stack.yaml

```diff
@@ -1,39 +1,34 @@
-resolver: nightly-2019-09-21 # Last GHC 8.6.5
+resolver: nightly-2020-01-31
```

Comment:
We cant build hie with `ghc-8.2.2`, if we use it in `stack.yaml` we should remove it from azure ci.
I think the plan was keeping ghc-8.6.5 in `stack.yaml` until `ghc-8.8.3` lands.

Reply:
Can we get azure to use `stack-8.6.5.yaml` instead? Which makes it clear

Reply:
mmm, there already is a job that uses `stack-8.6.5.yaml`. We can remove the one that uses `stack.yaml`